packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -46,7 +53,51 @@ contract FeeCurrencyWhitelist is
    * @return Patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 1, 0);
+    return (1, 1, 2, 0);
+  }
+
+  // removed with address zero
+  function setAdaptor(address token, address adaptor) external onlyOwner {
+    adapters[token] = adaptor;
+    emit AdaptorSet(token, adaptor);
+  }
+
+  // TODO make external
+  function getAdaptor(address underlyingToken) public view returns(address){
```

Comment:
Can we unify everything to Adapters rather than adaptors?

Reply:
thanks for doing that!